Dockerfile.orch-tester

```
@@ -0,0 +1,35 @@
+FROM livepeerci/build AS builder
+
+RUN apt install -y wget tar
+
+WORKDIR /root
+RUN wget -qO- https://storage.googleapis.com/lp_testharness_assets/official_test_source_2s_keys_24pfs_30s_hls.tar.gz | tar xvz -C .
+
+COPY go.mod go.mod
+COPY go.sum go.sum
+
+RUN go mod download
+
+ARG version
+RUN echo $version
+
+COPY . .
+
+RUN go build -ldflags="-X 'github.com/livepeer/stream-tester/model.Version=$version' -X 'github.com/livepeer/stream-tester/model.IProduction=true'" -tags mainnet cmd/orch-tester/orch_tester.go
+
+FROM debian:stretch-slim
+
+RUN apt update \
+  && apt install -y ffmpeg ca-certificates \
+  && apt clean && apt autoclean
+RUN ffmpeg -version
+RUN update-ca-certificates
+
+WORKDIR /root
+ENV PKG_CONFIG_PATH /root/compiled/lib/pkgconfig
+
+COPY --from=builder /root/compiled /root/compiled/
+COPY --from=builder /root/official_test_source_2s_keys_24pfs_30s_hls official_test_source_2s_keys_24pfs_30s_hls
+COPY --from=builder /root/orch_tester orch_tester
+
+CMD /root/orch_tester -v 6
```

Consider making this an `ENTRYPOINT` + `CMD` instead:
```suggestion
ENTRYPOINT ["/root/orch_tester"]
CMD ["-v", "6"]
```

This will allow the `orch_tester` to be run with custom arguments more easily, which will make it easier for Os to use it locally as well (e.g. `docker exec livepeer/orch-tester -- my custom args` instead of having to repeat the `/root/orch_tester` bit)

Also, maybe the `-v 6` bit should be specified as an environment variable instead like `ENV OT_V 6`, so its more easily overridable through the env but not automatically overridden by custom commands. That would be:
```suggestion
ENV OT_V 6
ENTRYPOINT ["/root/orch_tester"]
```
This also guarantees that setting a custom `OT_V` env still works, since I believe CLI args have precedence over env in our flag parsing lib 🤔

---

Oh, maybe for the `-v` flag to work as an env var you will need to do sth like this? 
 - https://github.com/livepeer/task-runner/blob/71f66c7621439b3eb0350f63f2c5db7ea594217b/cmd/task-runner/task-runner.go#L66-L67 
 - https://github.com/livepeer/task-runner/blob/71f66c7621439b3eb0350f63f2c5db7ea594217b/cmd/task-runner/task-runner.go#L77

(and if you do that, maybe we don't even need this default value in the Dockerfile. We can just set the default value of that custom flag to be `6`)

---

Right, update `-v` flag to the same way we use it in `go-livepeer` (a937c5a5b4949c50ef934de551735c336d8b26f8)

---

Updated ENTRYPOINT in e2ac4b42571044685e56043314e0c32d852dd9eb